ossdbtoolsservice/object_explorer/routing/mysql_routing.py

```diff
@@ -45,6 +45,9 @@ def _get_node_info(
     # Generate the object metadata
     metadata = ObjectMetadata(node.urn, None, type(node).__name__, node.name, None)
 
+    if hasattr(node, '_dbname') and node._dbname:
```

Comment:
Why is this required?

Reply:
If node has dbname attribute, pass into metadata as schema name. dbname and schema name are the same in MySQL. Added comment in code as well